pkg/agent/bakerapi.go

```diff
@@ -78,7 +78,9 @@ func (agentBaker *agentBakerImpl) GetNodeBootstrapping(ctx context.Context, conf
 		e := toggles.NewEntityFromNodeBootstrappingConfiguration(config)
 		imageVersionOverrides := agentBaker.toggles.GetLinuxNodeImageVersion(e)
 		if imageVersion, ok := imageVersionOverrides[string(distro)]; ok {
```

Comment:
we can probably simplify this

```suggestion
		if imageVersion := imageVersionOverrides[string(distro)]; imageVersion != "" {
		    nodeBootstrapping.SigImageConfig.Version = imageVersion
		}
```